### ISSUE TITLE
Menu bar not visible in Ubuntu 18.10

### ISSUE THREAD

**Issue by markojurkovic** [Foundry376/Mailspring#1258]
Due to the new default Yaru theme in Ubuntu 18.10 and also due to Electron, Mailspring menu bar is not visible on the stock Ubuntu 18.10 with the default theme (when the window is in focus). This is a known issue with Electron, and has been resolved on the Electron side here:  electron/electron#15194. (This issue also affected some other Electron-based apps.)

Now Mailspring should be rebuilt with the latest Electron version to fix this issue.

##### What operating system are you using?

Ubuntu 18.10

##### What version of Mailspring are you using?

1.5.3

##### Is this an email sync or connection issue?

No

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?

No

##### Is the issue reproducible with a particular attachment, message, signature, etc?

![screenshot from 2018-12-04 20-37-41](https://user-images.githubusercontent.com/9203900/49468537-70746000-f805-11e8-9f0e-982eb8a2f641.png)

![screenshot from 2018-12-04 20-45-03](https://user-images.githubusercontent.com/9203900/49468625-ac0f2a00-f805-11e8-8807-e258a81bf6e9.png)

With window out of focus:

![screenshot from 2018-12-04 20-46-57](https://user-images.githubusercontent.com/9203900/49468802-2cce2600-f806-11e8-8653-b31f0018d39e.png)

**Comment by mmstick** [comment]
This also happens on Pop!_OS with the Pop theme, so it's not limited to Yaru.

**Comment by Al2Me6** [comment]
Duplicate of #1158

**Comment by bengotow** [comment]
Hey folks - thanks for reporting this. It looks like we definitely need to move Mailspring to Electron 4 to get this resolved. We're currently blocked on some printing bugs in Electron (if you try to print a multi-page email you only get the first page), but I've submitted a PR to backport a patch for that into Electron 4.x and hopefully we can get it fixed soon. Stay tuned 🙏

**Comment by leoescarpellin** [comment]
Just to notify,
Linux Mint 19.1 is happening the same thing, it is not possible to view the menus and tray icon

**Comment by bengotow** [comment]
Hey folks—v1.5.4 just shipped earlier today and now uses the latest version of Electron (4.0.0). Could you give the new version a spin and see if it addresses this issue?

**Comment by krupenja** [comment]
Yes, can confirm that this is fixed in 1.5.4

**Comment by github-maverick** [comment]
Problem persists on 'Adapta' theme.

**Comment by bengotow** [comment]
Hey folks! Thanks for following up - I'll take a look at the Adapta theme, sounds like we may need to create a separate issue to track that.
